docs/getting-started-with-react/getting-started-with-react.stories.mdx

```diff
@@ -87,25 +87,7 @@ For this example we are going to build a card that allows you to enter a topic
 and then search for a related gif.
 
 <Canvas>
-  <Story name="Gif Gift">
-    {() => {
-      const [topic, setTopic] = useState();
-      const [url, setURL] = useState();
-      return (
-        <Card title="Gif Gift">
-          <Content>
-            <InputText value={topic} onChange={setTopic} placeholder="Topic" />
-            <Button label="Search" fullWidth={true} onClick={gifSearch} />
-            <center>{url && <img src={url} />}</center>
-          </Content>
-        </Card>
-      );
-      async function gifSearch() {
-        setURL(loadingGIF);
-        setURL(await getGIF(topic));
-      }
-    }}
-  </Story>
+  <GifGift />
```

Comment:
By pulling this out into a component in `getGif.tsx` and removing the `Story` we no longer get the unnecessary nesting on Storybook and the functionality will work on the new docs site (it wasn't if I just pulled in this mdx file as is).

Behaviour before on the Storybook site:

![Screenshot 2024-12-16 at 1 11 40 PM](https://github.com/user-attachments/assets/4f317ae3-be3f-4ce8-9234-b866818aa604)


Behaviour now:

![Screenshot 2024-12-16 at 1 09 34 PM](https://github.com/user-attachments/assets/ef6d8bb2-6a75-4d76-9716-b24fb82e489b)

Reply:
Yesss thank you for doing this! I noticed this last week and wanted to do the same thing. 👍🏻 Everything works still in Storybook and in new docs site